### ISSUE TITLE
Add a button to restart channel sync

### ISSUE THREAD

**Issue by yukibtc** [nbd-wtf/obw#36]
Hi, some days ago my OBW started a channel sync but it wasn't completed and now I having issue to send payments. I receive this error: "Channel sync is not finished yet, please wait a bit while keeping an app open and then try again.".
But the channel sync is not running (at least there isn't the channel sync tag). 
So would be useful to have a button to restart the channel sync to avoid to wait for the next sync.

**Comment by Gravity2106** [comment]
Can you see anything in a Blockexplorer?

**Comment by yukibtc** [comment]
Why blockexplorer? The balances are synced. I talking about the lightning channel sync. I receive that error message when I try to send a lightning payment.

**Comment by Gravity2106** [comment]
Oh, my fault. I understood that you were sucked on the channel opening.

Yes, maybe an option to sync again or a log to see how is the status could be nice

**Comment by fiatjaf** [comment]
I am having a hard time wrapping my head around this.

**Comment by fiatjaf** [comment]
I can't reproduce it either. My payments are always succeeding.

**Comment by yukibtc** [comment]
> I can't reproduce it either. My payments are always succeeding.

Now, after a successful channel sync, also my payments works. 
My problem was when the lightning channel sync wasn't completed (not in progress but failed, without channel sync progress bar).

**Comment by fiatjaf** [comment]
Good point, I hadn't investigated the possibility of the channel sync failing at some point for mysterious reasons. Will investigate that.
